pandas/tests/frame/methods/test_rank.py

```diff
@@ -502,14 +503,13 @@ def test_rank_mixed_axis_zero(self, data, expected):
         result = df.rank(numeric_only=True)
         tm.assert_frame_equal(result, expected)
 
-    @pytest.mark.parametrize(
-        "dtype, exp_dtype",
-        [("string[pyarrow]", "Int64"), ("string[pyarrow_numpy]", "float64")],
-    )
-    def test_rank_string_dtype(self, dtype, exp_dtype):
+    def test_rank_string_dtype(self, string_dtype_no_object):
         # GH#55362
-        pytest.importorskip("pyarrow")
-        obj = Series(["foo", "foo", None, "foo"], dtype=dtype)
+        obj = Series(["foo", "foo", None, "foo"], dtype=string_dtype_no_object)
         result = obj.rank(method="first")
+        exp_dtype = "Int64" if string_dtype_no_object.na_value is pd.NA else "float64"
```

Comment:
Hmm why are these int64 for some types now? I think float is required for the return type to allow for some tiebreakers

Reply:
Ah, good point! But so that is an existing bug in the `ArrowExtensionArray` implementation then, because this test was already asserting Int64 (I just rewrote it a bit, but in the removed parametrization a few lines above there is already this dtype)

Reply:
Ah ok...interesting!

Reply:
Opened a separate PR to address this -> https://github.com/pandas-dev/pandas/pull/59768